health_check/cache/backends.py

```diff
@@ -7,7 +7,16 @@
 
 
 class CacheBackend(BaseHealthCheckBackend):
+    def __init__(self, backend=''):
+        super().__init__()
+        self.backend = backend or 'default'
```

Comment:
I believe it's more verbose to have the default value in the signature.

```suggestion
    def __init__(self, backend='default'):
        super().__init__()
        self.backend = backend
```